samples/meetings-attendance-report/nodejs/resources/attendance-summary-card.json

```diff
@@ -0,0 +1,185 @@
+{
+    "type": "AdaptiveCard",
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "version": "1.4",
+    "body": [
+      {
+        "type": "TextBlock",
+        "size": "Large",
+        "weight": "Bolder",
+        "text": "Summary"
+      },
+      {
+        "type": "ColumnSet",
+        "columns": [
+          {
+            "type": "Column",
+            "items": [
+              {
+                "type": "TextBlock",
+                "text": "Attended Participant",
+                "wrap": true
+              },
+              {
+                "type": "TextBlock",
+                "weight": "Bolder",
+                "text": "${participantCount}",
+                "wrap": true,
+                "size": "Large"
+              }
+            ],
+            "width": "auto",
+            "horizontalAlignment": "center"
+          },
+          {
+            "type": "Column",
+            "items": [
+              {
+                "type": "TextBlock",
+                "text": "Start and end time",
+                "wrap": true
+              },
+              {
+                "type": "TextBlock",
+                "weight": "Bolder",
+                "text": "${meetingStartAndEndTime}",
+                "wrap": true,
+                "size": "Large"
+              }
+            ],
+            "width": "auto",
+            "horizontalAlignment": "center"
+          },
+          {
+            "type": "Column",
+            "items": [
+              {
+                "type": "TextBlock",
+                "text": "Meeting duration",
+                "wrap": true
+              },
+              {
+                "type": "TextBlock",
+                "weight": "Bolder",
+                "text": "${meetingDuration}",
+                "wrap": true,
+                "size": "Large"
+              }
+            ],
```

Comment:
If 5 people have joined the meeting, how are we handling dynamically and show all of them ?

Reply:
Using $data property in Adaptive card. we are passing an object from csharp in template.expand method.